client/acl.go

```diff
@@ -10,18 +11,38 @@ import (
 
 //https://docs.splunk.com/Documentation/Splunk/8.0.4/RESTUM/RESTusing#Access_Control_List
 func (client *Client) GetAcl(owner, app, name string, resources ...string) (*http.Response, error) {
-	resourcePath := []string{"services", owner, app}
+	resourcePath := []string{"servicesNS", owner, app}
```

Comment:
This seemed to be incorrect in general, and caused issues trying to use it without changing, because it has owner and app, which is only valid for `servicesNS`.

This method didn't seem to be used anywhere else, based on a simple `grep`.

Reply:
Ok. Yeah, I guess we never ran into errors since it was not being used. `servicesNS` makes sense.